src/components/genesys-web-message/GenesysWebMessage.vue

```diff
@@ -15,14 +15,14 @@ import { Component, Prop } from 'vue-property-decorator'
  */
 @Component({})
 export default class GenesysWebMessage extends Vue {
-  @Prop({ default: '' }) readonly genesysURL!: string
+  @Prop({ default: null }) readonly genesysURL!: string
 
-  @Prop({ default: '0' }) readonly environmentKey!: string
+  @Prop({ default: null }) readonly environmentKey!: string
 
   @Prop({ default: '0' }) readonly deploymentKey!: string
```

Comment:
You can default this one to null also, thus simplifying the check on line 25.

Reply:
Oh shoot - this is good if the parent provides a null value, but actually that key is a string from 1Password / config file, and we set it to a value of "0" to disable things, so you were absolutely correct to check it for `!== '0'`.

As for the URL and the environment props, well, I'm not going to set those to "0" in 1Password to disable this feature (when setting the deployment key to 0 is enough)... But I can't set those to "" either as 1Password/CD scripts will see that as a key mismatch between environments... So... 

Maybe the best solution is a bit of both:
1. default the props to null and ensure they are not falsy (as above)
2. verify that the deployment key is not "0"

Sorry, and please and thanks.

Reply:
Looks good.